docs/release-notes/index.md

```diff
@@ -21,6 +21,71 @@ To check for security updates, go to [Security announcements for the Elastic sta
 % ### Fixes [logstash-next-fixes]
 % *
 
+## 9.1.0 [logstash-9.1.0-release-notes]
+
+### Features and enhancements [logstash-9.1.0-features-enhancements]
+
+* Significantly improves write speeds to the persistent queue (PQ) when a pipeline's workers are caught up with already-written events [#17791](https://github.com/elastic/logstash/pull/17791)
+* Eliminated log warning about unknown gauge metric type when using pipeline-to-pipeline. [#17721](https://github.com/elastic/logstash/pull/17721)
+* Improve plugins remove command to support multiple plugins [#17030](https://github.com/elastic/logstash/pull/17030)
+* Deprecated the persistent queue setting `queue.checkpoint.interval`[#17759](https://github.com/elastic/logstash/pull/17759), which was found to have no effect. This will be removed in a future Logstash release.
+* Logstash now ships with JRuby 9.4.13.0 to leveragle latest features and improvements in the 9.4 series [#17696](https://github.com/elastic/logstash/pull/17696)
+* Enhanced keystore validation to prevent the creation of secrets in an invalid format [#17351](https://github.com/elastic/logstash/pull/17351)
+
+##### ES|QL Support in Elasticsearch Input (Technical Preview)
+
+* Support for using ES|QL queries in the [Elasticsearch input](https://www.elastic.co/guide/en/logstash/current/plugins-inputs-elasticsearch.html) to add improved flexibility when ingesting data from Elasticsearch is now in Technical Preview. 
```

Comment:
Not sure that linking to the highest level of the Elasticsearch input adds value is such a specific piece of content. Suggest either deep linking to new docs, or removing the link.

And/or adding a link to the PR where the work was done.